packages/atomic/src/global/global.pcss

```diff
@@ -97,4 +97,8 @@
       @apply font-bold border-2 border-primary;
     }
   }
+
+  .max-w-snippet {
+    max-width: 30ch;
+  }
```

Comment:
Hmm, was this class being referenced but did not exist?
Is the class referenced in multiple places to be in global, or can it be scoped to the breadcrumb pcss file?

Reply:
Yes, from the jira:

> Jean-Felix Thibodeau
4 days ago

For that one, I defined the global .max-w-snippet class but somehow it didn’t push/merge correctly. The fix is to set that class in global.pcss as having a width of 30ch

The text-overflow is already handled, yes.

I can look to move this to breadcrumb pcss file, see if it works. Indeed, it seems it's not used anywhere else.